pymatgen/entries/MPCompatibility.yaml

```diff
@@ -2,55 +2,46 @@ Name: MP
 Advanced:
   UCorrections:
     O:
-      Mn: -1.68085015096   #Fit to MnO, Mn3O4 and MnO2 (BURP:-1.687)
-      Fe: -2.733           #Fit to FeO and Fe2O3 (Fe3O4 probably wrong)
-      Co: -1.874           #Fit to CoO, Co3O4 (BURP:-1.751)
-      Cr: -2.013           #Fit to Cr2O3 (CrO3 missing) (BURP: -2.067)
-      Mo: -3.531           #Fit to MoO3 and MoO2 (BURP: -2.668)
-      W:  -4.351           #Fit to WO2 and WO3 (BURP: -2.762)
-      V:  -1.682           #Fit to V2O3 and V2O5 (VO2 fit is way off) (BURP: -1.764)
-      Ni: -2.164           #Based on burp version as of Feb 28 2011
+      V: -1.634860584846134
```

Comment:
We probably don't need so many sig figs. The corrections are definitely not good sub 1 meV, so let's round to that for now unless someone has a different opinion

Reply:
I agree with rounding to 1 meV

Reply:
Does this also apply to the correction errors? Rounding to 3 decimal places often results in only 1 sigfig (error = 0.00x), at most 2.